DCOS/templates/preprovision/extensions/preprovision-agent-windows/v1/preprovision-agent-windows-credentials.ps1

```diff
@@ -0,0 +1,13 @@
+$MESOS_SERVICE_DIR = Join-Path $env:SystemDrive "DCOS\mesos\service"
```

Comment:
Rename this script from `preprovision-agent-windows-credentials.ps1` to `preprovision-agent-windows-mesos-credentials.ps1`